packages/backend/src/services/fetch-instance-metadata.ts

```diff
@@ -156,7 +156,7 @@ async function fetchFaviconUrl(instance: Instance, doc: DOMWindow['document'] |
 	const url = 'https://' + instance.host;
 
 	if (doc) {
-		const href = doc.querySelector('link[rel="icon"]')?.getAttribute('href');
+		const href = doc.querySelector('link[rel="icon"], link[rel="shortcut icon"]')?.getAttribute('href');
```

Comment:
I think it should parse the attr value or it can still be broken. https://html.spec.whatwg.org/multipage/semantics.html#attr-link-rel

e.g.

```suggestion
		const links = Array.from(doc.querySelectorAll('link[rel*="icon"])).reverse();
		const link = links.find(link => link.relList.contains('icon'));
		const href = link?.href;
```

or

```suggestion
		const href = doc.querySelector('link[rel="icon"], link[rel^="icon "], link[rel$=" icon"], link[rel*=" icon "]')?.href;
```

(I'm sorry but these codes were not tested.)